### PR TITLE
Update mdb-create-edit-device-groups.md

### LINKED ISSUE #12027: Missing step in instructions
Under create a new device group on step two, there is now a Configuration management option that has to be picked first and then Device Configuration.

image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7a6e4ee4-97f2-9d40-5a92-4580fa0d25ac
* Version Independent ID: 7a6e4ee4-97f2-9d40-5a92-4580fa0d25ac
* Content: [Device groups in Microsoft Defender for Business](https://learn.microsoft.com/en-us/microsoft-365/security/defender-business/mdb-create-edit-device-groups?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-business/mdb-create-edit-device-groups.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-business/mdb-create-edit-device-groups.md)
* Service: **microsoft-365-security**
* Sub-service: **mdb**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12027

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-business/mdb-create-edit-device-groups.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Currently, in Defender for Business, you can create a new device group while you
 
 1. Go to the Microsoft 365 Defender portal ([https://security.microsoft.com](https://security.microsoft.com)) and sign in.
 
-2. In the navigation pane, choose **Device configuration**. 
+2. In the navigation pane, choose **Configuration Management** and select **Device configuration**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-business/mdb-create-edit-device-groups.md]:
```suggestion
2. In the navigation pane, choose **Configuration management** and select **Device configuration**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**